<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficLight</class>
 <widget class="QWidget" name="TrafficLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrafficLight</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>421</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>115</number>
      </property>
      <property name="bottomMargin">
       <number>115</number>
      </property>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>&amp;Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>S&amp;top</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="greenRadio">
        <property name="text">
         <string>&amp;Green</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="yellowRadio">
        <property name="text">
         <string>&amp;Yellow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="redRadio">
        <property name="text">
         <string>&amp;Red</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
